uportal-war/src/main/java/org/jasig/portal/rest/PortletsRESTController.java

```diff
@@ -55,24 +62,22 @@
 @Controller
 public class PortletsRESTController {
 
+    @Autowired
     private IPortletDefinitionRegistry portletDefinitionRegistry;
+
+    @Autowired
     private IPortletCategoryRegistry portletCategoryRegistry;
-    private IPersonManager personManager;
 
     @Autowired
-    public void setPortletDefinitionRegistry(IPortletDefinitionRegistry portletDefinitionRegistry) {
```

Comment:
That's a good question.  I get the sentiment or rationale, but I'd say it isn't.

 * This is a @Controller class;  it probably doesn't have direct callers/clients in uPortal (probably shouldn't)
 * The bean-style setters are noise in the file -- they're what I have to scroll through to see code I care about
 * One of the great lessons-learned in the evolution from channels to portlets was that it's tremendously difficult and limiting to hold out every single type and method signature in uPortal's own class loader as some kind of public API with a commitment to backwards compatibility;  we are at a point where if you have Java code that directly calls/implements/inherits from Java code distributed with uPortal under uportal-war/src/main/java, it should probably be contributed.  (Exceptions might be the pluggable interface-based abstractions like IPersonAttributesDAO.)